packages/twenty-front/src/modules/object-record/record-table/hooks/usePendingRecordId.ts

```diff
@@ -0,0 +1,13 @@
+import { useRecoilState } from 'recoil';
+
+import { pendingRecordIdState } from '@/object-record/record-table/states/pendingRecordIdState';
+
+export const usePendingRecordId = () => {
+  const [pendingRecordId, setPendingRecordId] =
```

Comment:
I don't think this is a good pattern to introduce a hook to wrap useRecoilState.
The issue is that this hook will subscribe to the state (and trigger re-renders) even if you are only interested in the setter. I would call directly useRecoilState, useRecoilValue, useSetRecoilState where needed in your code